src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -116,9 +116,13 @@ def create_control_mode_menu(
         )
 
         if IndividualRobotMode.AI in available_control_modes:
-            control_mode_menu.setCurrentIndex(2)
+            control_mode_menu.setCurrentIndex(
+                control_mode_menu.findText(IndividualRobotMode.AI.name)
```

Comment:
Interesting, `name` returns the index?

Reply:
no I think name returns the text name, but findText returns the index